sites/public/src/components/applications/ApplicationMultiselectQuestionStep.tsx

```diff
@@ -85,27 +91,67 @@ const ApplicationMultiselectQuestionStep = ({
     return getAllOptions(question, applicationSection)
   }, [question])
 
+  const body = useRef(null)
+
   const onSubmit = (data) => {
-    const body =
-      questionSetInputType === "checkbox"
-        ? mapCheckboxesToApi(data, question, applicationSection)
-        : mapRadiosToApi(data.application[applicationSection], question)
-    if (questions.length > 1 && body) {
+    if (verifyAddressStep === 0) {
+      body.current =
+        questionSetInputType === "checkbox"
+          ? mapCheckboxesToApi(data, question, applicationSection)
+          : mapRadiosToApi(data.application[applicationSection], question)
+    }
+
+    // Verify address on preferences
+    if (question?.options.some((item) => item?.validationMethod)) {
```

Comment:
Sorry if this wasn't clear in the ticket. We want this address validation done on all preferences that collect address. Not just ones that have a "validationMethod" setup. So even existing preferences that collect address will start getting this address verification step